server/db/index.js

```diff
@@ -1,8 +1,17 @@
 require('dotenv').config()
 const mongoose = require('mongoose')
 
+let db_url = 'mongodb'
+if (process.env.DB_SRV != "false") {
+    db_url += '+srv'
+}
+let db_url_log = db_url+`://XXXX:XXXX@${process.env.DB_ADDRESS}/${process.env.DB_NAME}?retryWrites=true&w=majority`
```

Comment:
I'm just curious. Does this `XXXX:XXXX` work as a wildcard?

Reply:
That's to hide the username/password.